index.js

```diff
@@ -333,6 +333,17 @@
 //. <script src="vendor/sanctuary-bundle.js"></script>
 //. ```
 //.
+//. The bundle can also be included via:
+//. ```html
+//. <script src="https://sanctuary.js.org/vendor/sanctuary-bundle.js"></script>
+//. ```
+//.
+//. Installation of the bundle through jsdelivr:
+//. ```html
+//. <script src="https://cdn.jsdelivr.net/gh/sanctuary-js/sanctuary@3.0.0/dist/bundle.js"></script>
```

Comment:
Perhaps we should use `X.Y.Z` as a placeholder, as we do above:

```suggestion
//. <script src="https://cdn.jsdelivr.net/gh/sanctuary-js/sanctuary@X.Y.Z/dist/bundle.js"></script>
```

In fact, this approach could supersede the `curl` approach.